05-Token-Renewal/README.md

```diff
@@ -1,6 +1,6 @@
 # Auth0 React Token Renewal
 
-This sample demonstrates how to silently renew `access_token`s in a React application with Auth0. The sample uses the create-react-app.
+This sample demonstrates how to renew `access_token`s in a React application with Auth0 using `checkSession`. For more information, read [our reference documentation](https://auth0.com/docs/libraries/auth0js/v8#using-checksession-to-acquire-new-tokens). The sample uses the create-react-app.
 
 ## Getting Started
```

Comment:
Maybe we can use https://auth0.com/docs/libraries/auth0js#using-checksession-to-acquire-new-tokens that links to the latest version,.